content/id/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Poseidon-Firmament - Sebuah Penjadwal Alternatif
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Rilis saat ini dari Penjadwal Poseidon-Firmament adalah rilis <code> alfa </code>.**
+
+Penjadwal Poseidon-Firmament adalah penjadwal alternatif yang dapat digunakan bersama penjadwal Kubernetes bawaan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Pengenalan	
+
+Poseidon adalah sebuah layanan yang berperan sebagai pemersatu antara [Penjadwal Firmament](https://github.com/Huawei-PaaS/firmament) dengan Kubernetes. Penjadwal Poseidon-Firmament menambah kapabilitas penjadwal Kubernetes saat ini. Penjadwal ini menggabungkan kemampuan penjadwalan berbasis aliran jaringan grafis baru bersama penjadwal Kubernetes bawaan. Penjadwal Firmament memodelkan beban kerja dan kelompok sebagai jaringan aliran dan menjalankan optimisasi aliran biaya-minimum kepada jaringan ini untuk membuat keputusan penjadwalan.
+
+Penjadwal ini memodelkan masalah penjadwalan sebagai optimasi berbasis kendala atas grafik jaringan aliran. Hal ini dicapai dengan mengurangi penjadwalan ke masalah optimisasi biaya-minimum aliran-maks. Penjadwal Poseidon-Firmament secara dinamis memperbaiki penempatan beban kerja.
+
+Penjadwal Poseidon-Firmament berjalan bersamaan dengan penjadwal Kubernetes default sebagai penjadwal alternatif, sehingga beberapa penjadwal dapat berjalan secara bersamaan.
```

Comment:
```suggestion
Penjadwal Poseidon-Firmament berjalan bersamaan dengan penjadwal Kubernetes _default_ sebagai penjadwal alternatif, sehingga beberapa penjadwal dapat berjalan secara bersamaan.
```

Reply:
i change it into `bawaan` for consistency